docs/specs/src/hwp_1_intro.md

```diff
@@ -0,0 +1,212 @@
+challenge of anti-fragility in such contexts is composable sub-systems,
+in which the composition comes out of a grammatics that embodies the
+actual dimensionality of the problem subdomains (i.e. their geometry),
+and by which agents in that context can react powerfully to
+perturbations because the available composability is dimensionally
+aligned.
+
+The core axiom (though not explicitly stated as such) of the Byzantine
+Generals Problem is that coordination starts AFTER "consensus on state",
+i.e. that the Lieutenants can't execute their plan until they have
+followed the consensus algorithm and arrived at single data reality[^3].
+This axiom leads system designers to figure out how to implement
+machinery for **Global Consensus**. Our axioms lead us, instead, to
+implement tooling for **Scaling Consent** as an alternate solution to
+the Byzantine Generals Problem.
+
+From Global Consensus to Scaled Consent
+---------------------------------------
+
+In distributed systems, we believe that it is absolutely fundamental to
+understand that every action taken by an agent in any social context,
+happens because that agent has crossed a confidence threshold of some
+sort, from its own point of view, that the given action is appropriate
+to take. Stated another way: agentic assessment of the social context
+and its coherence allows agents to act. This is always true, be it as
+solved in the Byzantine Generals Problem problem or by Blockchain
+consensus solutions.
+
+We also start from the understanding that social spaces are inherently
+uncertain. Thus, coordination/collaboration is never about deterministic
+certainty but simply about the capacity to remove sufficient uncertainty
+to provide confidence for action, which is always contextual. Such
+confidence indicates **social coherence**. This notion of social
+coherence is the single most important design goal of Holochain: to
+create the tooling that in contextually appropriate ways leads to
+increasing social coherence.
+
+Given the above, we propose a very simple approach to creating tooling
+capacity for building increasing certainty: **enable validated global
+visibility, on demand, of local state**. In this approach, we
+distinguish between authorship, which is about local state changes
+initiated by agents, and authority, which is about distributing the
+responsibility of validating and making visible those state changes
+across the participants in the network. This approach requires that we:
+
+1.  Ensure that all agents know the "ground-rules". i.e. offer a
+  framework for composing lots of small units of social agreement in
+  which players can add deterministic elements of certainty into
+  their interactions.
+
+2.  Ensure that all agents can *reliably* see what's going on. i.e.
+  offer a framework for adding to and querying a collectively held
+  database in which there is a knowable level of confidence in its
+  reliability and integrity that can also be graduated to fit the
```

Comment:
"that can be graduated to fit the circumstances' - what does that mean?

Reply:
It means in the sense of the definition: "change (something, typically color or shade) gradually or step by step."  That's a less common meaning of the word, maybe something else would be better?

Reply:
I understand "graduated", but the "knowable level of confidence in its reliability and integrity" reads as very defined whereas "graduated to fit the circumstances" by itself without further details waters down the strictness IMO

Reply:
Maybe if it adds something like "graduated to fit the circumstances, ranging from an arbitrarily low level of confidence to the highest possible level certainty"

Reply:
Added what you suggested @maackle